src/crimson/os/seastore/object_data_handler.cc

```diff
@@ -339,7 +342,9 @@ overwrite_ops_t prepare_ops_list(
 	  [&region, &to_remap](auto &r) {
 	    interval_set<uint64_t> range;
 	    range.insert(r.pin->get_key(), r.pin->get_length());
-	    if (range.contains(region.addr, region.len)) {
+	    if (range.contains(region.addr, region.len) &&
+		static_cast<lba_manager::btree::BtreeLBAMapping*>(
+		r.pin.get())->get_map_val().refcount == 1) {
```

Comment:
Let's introduce a `LBAMapping::is_clone()` interface, encapsulating the implementation details here.

Reply:
Done.